app/controllers/v2/content_items_controller.rb

```diff
@@ -22,7 +22,7 @@ def linkables
     end
 
     def embedded
-      results = GetHostContentService.new(
```

Comment:
oh I see the presenter and query use `GetEmbedded`, I thought we'd changed them 😅  - I think it would be worth doing a rename because we do need code to actually get embedded content versus getting host documents  (e.g. `EmbeddedContentFinderService`) and it could get confusing quickly..

Reply:
PR to do the rename is here https://github.com/alphagov/publishing-api/pull/2991